miradar_node:
   ros__parameters:
      devicename: /dev/ttyACM0
      sensor_mode: 0
      max_distance: 4.0
      min_distance: 0.3
      alarm_distance: 1.0
      max_dB: -30
      min_dB: -74
      distance_div: 64
      angle_div: 45
      tx_power: -7
      hpf_gain: 1
      pga_gain: 1
      duration: 200
      scan_mode: 0